index.js

```diff
@@ -217,6 +217,11 @@ function fromJsonObject (that, object) {
 if (Buffer.TYPED_ARRAY_SUPPORT) {
   Buffer.prototype.__proto__ = Uint8Array.prototype
   Buffer.__proto__ = Uint8Array
+  if (typeof global.Symbol !== 'undefined' && Symbol.species &&
```

Comment:
I would prefer `typeof Symbol` instead of `typeof global.Symbol`. Reason being: it might fail in odd environment like NW.js that have dual global objects.